internal/serverclient/token.go

```diff
@@ -62,5 +68,39 @@ func TokenFromContext(ctx context.Context) string {
 	}
 
 	return raw.(string)
+}
+
+const (
+	// tokenMagic is used as a byte sequence prepended to the encoded TokenTransport to identify
+	// the token as valid before attempting to decode it. This is mostly a nicity to improve
+	// understanding of the token data and error messages.
+	tokenMagic = "wp24"
```

Comment:
Not ideal to duplicate the token magic, but not the worst.

Reply:
Going to leave the duplication for now, refactoring for this one thing is more ugly.